app/controllers/project.py

```diff
@@ -128,7 +130,7 @@ def post(self):
                         message='Internal Server Error'), 500
 
             # create a billing invoice on project creation
-            new_invoice = BillingInvoice(project_id=project.id)
+            new_invoice = BillingInvoice(id="CC"+str(time.strftime("%y", time.localtime()))+"-"+str(uuid.uuid4())[:8], project_id=project.id)
```

Comment:
Hey @Awath-Abdat, do you mind elaborating on what this line does., why the changes?

Reply:
A custom ID was needed so every time a new invoice is created, i made the custom ID to be sent along during the creation of the billing invoice record. I didn't find a way of making the custom ID of the format CCYY-1a2b3c4d to be created by default on postgres db when the record is being created.

Reply:
OOh makes sense, alright, so create a different field o=in the invoice table for the custom id in the DB. An that's for display

Reply:
@Awath-Abdat could you spare a moment and work on this feedback by creating a different field in the invoice table to display the formatted invoice id.

Reply:
Am going to make the change, so i should create a seperate field for this new invoice ID instead of making changes to the original UUID ID which we were using

Reply:
You can revert the invoice_id field back to uuid  and create a new fields called invoice_display_id that will be displayed in the emailed invoice. We shall maintain the invoice_id in UUID form. Make sense? @Awath-Abdat

Reply:
Is it possible to push these changes before the demo @Awath-Abdat .

Reply:
I have adjusted and made the changes, i was in a paper today morning so worked on it after the paper